pkgs/tools/archivers/p7zip/default.nix

```diff
@@ -46,13 +52,20 @@ stdenv.mkDerivation rec {
 
   NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-error=c++11-narrowing";
 
-  meta = {
+  passthru.updateScript = ./update/update.sh;
```

Comment:
Hm, shouldn't that file be also among changes?

Reply:
:facepalm: 

added